ouroboros-consensus-cardano/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 # Changelog entries
 
+## 0.6.2.0
+
+### Non-Breaking
+
+- Add `GetConstitutionHash` ledger query
+
```

Comment:
CHANGELOG.md is not modified manually, you'll need to use `scriv` to add changelog entries in consensus, see this reference doc: https://github.com/input-output-hk/ouroboros-consensus/blob/main/docs/website/contents/for-developers/ReleaseProcess.md#installing-scriv

Reply:
Thanks!